packages/core/hardhat.config.ts

```diff
@@ -72,7 +71,7 @@ task('deploy-system')
     ) => {
       // Throw an error if we're on the Hardhat network. This ensures that the `url` field is
       // defined for this network.
-      if (!isHttpNetworkConfig(hre.network.config)) {
+      if (!('url' in hre.network.config)) {
```

Comment:
Just removed the `isHttpNetworkConfig` because it required another type called `NetworkConfig` which conflicted with the new name for the parsed config which is `NetworkConfig`.